impl/pom.xml

```diff
@@ -84,6 +84,8 @@
                   <artifact>org.apache.commons:commons-lang3</artifact>
                   <includes>
                     <include>org/apache/commons/lang3/StringUtils**</include>
+                    <include>org/apache/commons/lang3/CharSequenceUtils**</include>
+                    <include>org/apache/commons/lang3/ArrayUtils**</include>
```

Comment:
Roaster actually doesn't use these classes, so I don't think this is the right fix. 

I think the correct approach should be to shade the used classes under a different package name to avoid conflicts

Reply:
The commons-lang3 library is already shaded properly, as the org.apache package is shaded, there is no need to shade org.apache.commons-lang3.
I was referring to my usage of StringUtils from roaster and possibly other users of roaster who could encounter the same 'crash'.
In the end you can keep a partial shading of commons-lang3 like this because it's intended -in this case the original commons-lang3 must be used by clients who need it-, or else you can remove the listed includes to shade commons-lang3 as a whole.
I think it would make more sense to shade commons-lang3 as a whole, but maybe you have your reasons for doing a partial shading with a list of includes.
For example, if the roaster project happens to introduce calls to StringUtils.equalsAny() or StringUtils.startsWithAny(), the issue could go unnoticed until it crashes, and then you would have to add the required includes one by one until there is no more crash.

Reply:
@nbauma109 absolutely, but in that case I'd expect a test in the `tests/` that covers the code using these methods

Reply:
Because the StringUtils in Roaster shouldn't be used outside the library I think shading into another package makes more sense

Reply:
> The commons-lang3 library is already shaded properly, as the org.apache package is shaded, there is no need to shade org.apache.commons-lang3.

Ah, you're right, I missed that setting. Anyway, again, you shouldn't be using the shaded classes outside the Roaster library.

Reply:
> I think it would make more sense to shade commons-lang3 as a whole, but maybe you have your reasons for doing a partial shading with a list of includes.

The main reason is to keep the JAR small. Including all classes from `commons-lang3` (and `commons-text` added transitively) increases the final JAR by a bit more than 1 Mb, which is kinda annoying since the majority of the classes from those libraries are not used.